examples/tokenvm/storage/storage.go

```diff
@@ -19,7 +19,6 @@ import (
 	"github.com/ava-labs/hypersdk/state"
 
 	smath "github.com/ava-labs/avalanchego/utils/math"
-
```

Comment:
nit: undo these diffs that aren't related

Reply:
reverted but these are either done automatically or when the `./scripts/test.lint.sh` is ran.

Reply:
Confirmed that this is the `./scripts/test.lint.sh` script doing this